
# HG changeset patch
# User Stephan Bosch <stephan@rename-it.nl>
# Date 1414713028 -3600
# Node ID 3e924b1b6c5c0199f1b97cb5782f52e6348a9840
# Parent  80b47902cf40053cf13d8edbf034d0e4f8a4da07
After make distclean the distributed tarball would fail to recompile.

diff -r 80b47902cf40 -r 3e924b1b6c5c Makefile.am
--- a/Makefile.am	Thu Oct 30 23:27:42 2014 +0100
+++ b/Makefile.am	Fri Oct 31 00:50:28 2014 +0100
@@ -2,7 +2,10 @@
 DOCS = doc
 endif
 
-SUBDIRS = src $(DOCS)
+SUBDIRS = \
+	. \
+	src \
+	$(DOCS)
 
 ACLOCAL_AMFLAGS = -I m4
 
@@ -11,6 +14,7 @@
 	examples \
 	COPYING.LGPL \
 	ChangeLog \
+	is-tagged.py \
 	update-version.sh
 
 dist-hook:

